### PR TITLE
osa - add overrides

### PR DESCRIPTION
**Pull request by mjudeikis**
add test overrides file to manage those in our code repo, instead of release

### CODE REVIEW

**Comment by mjudeikis** [comment]
I think we can get rid of the release-3.10.yaml ci-operator config that you are adding in this PR and use the openshift-ansible ci-operator config directly.

**Comment by mjudeikis** [comment]
ping @kargakis @charlesakalugwu

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-upgrade.yaml [hunk 1]
```diff
@@ -148,6 +148,7 @@ objects:
           touch /tmp/shared/exit;
         fi; 
         cp -r /go/src/github.com/openshift/openshift-azure/_data /tmp/shared &>/dev/null
+        cp /go/src/github.com/openshift/openshift-azure/test/overrides /tmp/shared
```
Comment (kargakis) [inline ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-upgrade.yaml]: The overrides apply only in the conformance template. Please remove from elsewhere.

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml [hunk 2]
```diff
@@ -135,6 +135,7 @@ objects:
           touch /tmp/shared/exit;
         fi;
         cp -r /go/src/github.com/openshift/openshift-azure/_data /tmp/shared &>/dev/null
+        cp /go/src/github.com/openshift/openshift-azure/test/overrides /tmp/shared
```
Comment (kargakis) [inline ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml]: Remove

**Pull request merged**